lib/IR/GraphScheduler.cpp

```diff
@@ -144,8 +144,8 @@ class ChildMemSizeBasedScheduler : public Scheduler {
     // We don't model memory dependencies, but we still need to honor them.
     // Make sure the SaveNode happens after the last use of the output variable.
     if (auto *save = dyn_cast<SaveNode>(N)) {
-      Variable *output = save->getVariable();
```

Comment:
Given getVariable may return something that is not a variable now, I would replace it into getStorage instead of going through getOutput.

(I understand this may impact a lot of unittests, feel free to do it in a follow-up PR or file an issue ;))

Reply:
Thank you. Yes. I have this change locally.  I plan to change this to getOutput in a follow up PR.

Reply:
Actually, this line changes getVariable to getOutput, which supports both Variable and Placeholder.

Reply:
Yes. You are right. It should be getDestination().